.travis.yml

```diff
@@ -56,6 +56,8 @@ matrix:
       osx_image: xcode9.4
       env:
         - PYENV=3.6.1 MAC_WHEEL=1
+    - os: linux
+      arch: ppc64le
```

Comment:
I am not sure which CPython is provided, but 3.6 definitely is not the one.

Reply:
Should be fixed in https://github.com/vmprof/vmprof-python/pull/239/commits/0c6fdef0b47d888925d4f7acaedeea5987103d11